ipn/ipnstate/ipnstate.go

```diff
@@ -53,6 +53,9 @@ type Status struct {
 	// If nil, an exit node is not in use.
 	ExitNodeStatus *ExitNodeStatus `json:"ExitNodeStatus,omitempty"`
 
+	// DERPHomeRegionID is the current home DERP region ID.
+	DERPHomeRegionID int
```

Comment:
I guess we'll consider joining with DERPMap context where this gets surfaced in order to give them region name/code

Reply:
yeah. I figured it was wasteful to denormalize it here considering how often this is polled sometimes and how big the DERPMap can be